lib/commands/navigation.js

```diff
@@ -0,0 +1,27 @@
+import { errors } from 'appium-base-driver';
+import _ from 'lodash';
+import log from '../logger';
+
+
+var webdriver = require('selenium-webdriver'),
+	Key = webdriver.Key;
+
+let commands = {}
+
+commands.back = async function () {
+  var commandObject = {
+    name: 'sendkeys',
+    args: [Key.ESCAPE]
```

Comment:
Shouldnt the back command be `[Key.BACK]`?

Reply:
BACK doesn't exist ;)
https://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/Keys.html